Branch-SDK/src/main/java/io/branch/referral/util/BranchEvent.java

```diff
@@ -256,7 +256,7 @@ public boolean logEvent(Context context) {
         return isReqQueued;
     }
 
-    private class ServerRequestLogEvent extends ServerRequest {
+    public class ServerRequestLogEvent extends ServerRequest {
```

Comment:
Can you extract this out to its own file? I'd personally like to get rid of nested classes for easier project readability.